build/ChangeVersion.proj

```diff
@@ -28,12 +28,20 @@
       <ExpectedMatchCount>1</ExpectedMatchCount>
     </AssemblyInformationalVersion>
 
-    <VsixVersion Include="$(SolutionRoot)\src\Integration.Vsix\Manifests\VS201*\source.extension.vsixmanifest">
+    <!-- Set VSIX version in all VSIXes, including the embedded vsix -->
+    <VsixVersion Include="$(SolutionRoot)\src\Integration.Vsix\Manifests\VS201*\source.extension.vsixmanifest;$(SolutionRoot)\src\EmbeddedVsix\source.extension.vsixmanifest">
```

Comment:
The first part of the file pattern is the same (selects the main VSIX files).
The bit after the semi-colon is new (also select the embedded VSIX).